app_legacy/Community/Enums/ClaimFilters.php

```diff
@@ -34,13 +34,10 @@ abstract class ClaimFilters
     public const AllFilters = (1 << 13) - 1;
 
     // Filter to show all Complete Primary claims
-    public const AllCompletedPrimaryClaims = self::AllFilters & ~self::ActiveClaim
-                                            & ~self::DroppedClaim
-                                            & ~self::CollaborationClaim;
+    public const AllCompletedPrimaryClaims = self::AllFilters & ~self::ActiveClaim & ~self::DroppedClaim & ~self::CollaborationClaim;
 
     // Default filter is everything except Complete and Dropped claims
-    public const Default = self::AllFilters & ~self::CompleteClaim
-                                            & ~self::DroppedClaim;
+    public const Open = self::AllFilters & ~self::CompleteClaim & ~self::DroppedClaim;
```

Comment:
`AllOpenClaims` to be consistent with `AllCompletedPrimaryClaims`? Or maybe `AllActiveClaims` to also be consistent with the enum.

Reply:
Changed to `AllActiveClaims` in https://github.com/RetroAchievements/RAWeb/pull/1357/commits/ea2e99e57e2d9c9f2502dcc9f69bcc1cd212d022